docs/sphinx/source/conf.py

```diff
@@ -55,12 +55,15 @@ def __getattr__(cls, name):
     'sphinx.ext.viewcode',
     'sphinx.ext.intersphinx',
     'sphinx.ext.extlinks',
-    'numpydoc',
+    'sphinx.ext.napoleon',
     'sphinx.ext.autosummary',
+    'numpydoc',
```

Comment:
delete?

Reply:
It seems to render fine without it, and I suppose we can also remove a dependency, would probably also need to remove the numpydoc directive a few lines above above. Sure - your call. Let's do it. 🚀

Reply:
yes, numpydoc recommends against using it at the same time as napoleon. Also needs to be removed from setup.py